src/main/java/com/bankbazaar/test/controller/DepartmentController.java

```diff
@@ -0,0 +1,49 @@
+package com.bankbazaar.test.controller;
+
+import com.bankbazaar.test.entity.Department;
+import com.bankbazaar.test.service.DepartmentService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
```

Comment:
The url mapping should be like
@RestController
@RequestMapping("/departments")

Then the individual functions should just be the method. Ex POST on this url will signify create, get will signify fetch etc..

Reply:
I have implemented @RequestMapping as requested